### ISSUE TITLE
Edit steps to Epsilon-Delta proof

### ISSUE THREAD

**Issue by APEXCalculus** [APEXCalculus/APEXCalculusPTX#321]
In the steps leading to Equation (1.2.3) in 
[Example 1.2.6](https://opentext.uleth.ca/apex-calculus/sec_limit_def.html#ex_compute_lim4), we establish that x^2+x-1 could be negative, yet also state that
x^2+x-1 < 5 implies
1/(x^2+x-1) > 1/5. 

We need to add abs. values to some of these steps.

**Comment by sean-fitzpatrick** [comment]
Agreed! We usually don't teach this section, so I haven't looked carefully at the proofs.

We have the `\abs{}` macro for adding absolute values.

**Comment by sean-fitzpatrick** [comment]
I am just looking at this now: there is a bigger issue! On the interval [0,2], x^2+x-1 has a root!

That means there's a division by zero problem that perhaps you'd like to avoid. (On the other hand, I suppose it is true that infinity is greater than 1/5...)

The root is at (\sqrt{5}-1)/2 \approx 0.62.

I propose the following:

Since x approaches 1, we are safe to assume that x is between 3/4 and 5/4. (This interval contains 1 but does not contain the root.)

Then 9/16 \lt x^2 \lt 25/16. Adding the inequality 3/4\let x\leq 5/4, we get 21/16\lt x^2+x\lt 45/16.
Subtracting 1 gives 5/16\lt x^2+x-1 \lt 29/16.

Therefore, 16/29 \lt 1/(x^2+x-1) \lt 16/5.


So your delta is the minimum of 1/4 and 16/29 \varepsilon.

Not as pretty, but perhaps more educational? What do you think, @APEXCalculus ?

**Comment by sean-fitzpatrick** [comment]
I have changes for this done on a branch; pull request coming soon. If we want a different resolution to this problem, we can debate it there.
